components/Button.tsx

```diff
@@ -6,6 +6,9 @@ import styles from '@components/Button.module.scss';
 import LoaderSpinner from '@components/LoaderSpinner';
 
 const Button = (props: any) => {
+  if (props.disabled) {
+    return <button style={props.style} className={U.classNames(styles.button, styles.disabledButton)} children={props.children} disabled={props.disabled}/>;
```

Comment:
nit: might want to make sure prettier is enabled.

Reply:
good catch. Didn't have prettier enabled :) Fixed now.